boat_simulator/nodes/physics_engine/fluid_forces.py

```diff
@@ -72,9 +166,141 @@ def interpolate(self, attack_angle: Scalar) -> Tuple[Scalar, Scalar, Scalar]:
                     area is expressed in square meters (m^2).
         """
 
-        # TODO: Implement this method using `np.interp`.
+        lift_coefficient = np.interp(
+            attack_angle, self.__lift_coefficients[:, 0], self.__lift_coefficients[:, 1]
+        )
+        drag_coefficient = np.interp(
+            attack_angle, self.__drag_coefficients[:, 0], self.__drag_coefficients[:, 1]
+        )
+        area = np.interp(attack_angle, self.__areas[:, 0], self.__areas[:, 1])
+        return lift_coefficient, drag_coefficient, area
+
+    def draw_boat(ax, position, orientation):
```

Comment:
Does this function belong in this class?

Reply:
Where should it go?